templates/meta-release.j2

```diff
@@ -1,23 +1,23 @@
 {% if 'lts' in item.split('-') %}
-{%   for dist in metareleases_info | rejectattr('version', 'match', '^.*LTS$') %}
-{%     if dist.name.split(' ')[0] | lower in metareleases | map(attribute='name') | list %}
-{%       set metareleases = metareleases.remove({'name': dist.name.split(' ')[0] | lower}) %}
-{%     endif %}
+{%   for dist in metareleases.copy() %}
```

Comment:
hm... Looking a bit more.. Probably more "jinja2onic" to use `metareleases | list` than the `copy()` method..